client/src/main/java/io/pravega/client/stream/impl/ControllerImpl.java

```diff
@@ -243,9 +244,18 @@ public ControllerImpl(ManagedChannelBuilder<?> channelBuilder, final ControllerI
                                                                   .newBuilder().setScope(scopeInfo).setContinuationToken(token).build(), callback);
                         return callback.getFuture()
                                        .thenApply(x -> {
-                                           List<Stream> result = x.getStreamsList().stream()
-                                                                  .map(y -> new StreamImpl(y.getScope(), y.getStream())).collect(Collectors.toList());
-                                           return new AbstractMap.SimpleEntry<>(x.getContinuationToken(), result);
+                                           switch (x.getStatus()) {
+                                               case SCOPE_NOT_FOUND:
```

Comment:
We can add a log similar to the other apis with the pattern `log.warn(requestId, "Scope {} does not exist", scopeName);`
and a INFO log with `requestId` when it succeeds.

Reply:
the caller who made the exception will get the error and already logs all exceptions. Do we need the log here?

Reply:
I see this pattern for other apis.
https://github.com/pravega/pravega/blob/b21decf445312ff3cc6d89dbf7563bf80019b3eb/client/src/main/java/io/pravega/client/stream/impl/ControllerImpl.java#L206-L217